src/main/main.c

```diff
@@ -101,8 +107,12 @@ struct re {
 	struct list tmrl;            /**< List of timers                    */
 
 #ifdef HAVE_POLL
+#ifdef WIN32
+	WSAPOLLFD *fds;
+#else
```

Comment:
perhaps we should add a `HAVE_WSAPOLL` flag ?

it would be nice if the same codebase can be used to build for different
platforms, by setting flags.